docs/DeveloperGuide.md

```diff
@@ -282,14 +282,40 @@ _{Explain here how the data archiving feature will be implemented}_
-|----------|--------------------------------------------|------------------------------|------------------------------------------------------------------------|
-| `* * *`  | new user                                   | see usage instructions       | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person             |                                                                        |
-| `* * *`  | user                                       | delete a person              | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name        | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name         | locate a person easily                                                 |
+| Priority | As a …​                | I want to …​                                                                          | So that I can…​                                                 |
+|----------|------------------------|---------------------------------------------------------------------------------------|-----------------------------------------------------------------|
+| `* * *`  | volunteer coordinator  | create new events by specifying the name of the event                                 | refer to it in the future                                       |
+| `* * *`  | volunteer coordinator  | create new events by specifying the date and time of the event                        | check the date and time of an event in the future               |
+| `* * *`  | volunteer coordinator  | change the name of the event                                                          | update an event with the correct name                           |
+| `* * *`  | volunteer coordinator  | change the date and time of an event                                                  | update an event with the correct date and time                  |
+| `* * *`  | volunteer coordinator  | check the list of events through the app                                              | view all upcoming events                                        |
+| `* * *`  | volunteer coordinator  | check the date and time of an event                                                   | know when and what time an event will occur                     |
+| `* * *`  | volunteer coordinator  | delete volunteer opportunities                                                        | remove them when they are no longer needed                      |
```

Comment:
I think `delete volunteer events` is clearer in showing that you want to delete **events**

Reply:
yea agree, should be consistent in terminology

Reply:
or rather, just `delete events` to confuse the users less